custom-vu/src/main/kotlin/jces1209/vu/page/boards/view/ScrumBoardPage.kt

```diff
@@ -0,0 +1,20 @@
+package jces1209.vu.page.boards.view
+
+import jces1209.vu.page.boards.sprint.SprintPage
+import org.openqa.selenium.WebDriver
+import java.net.URI
+
+abstract class ScrumBoardPage(
+    driver: WebDriver,
+    uri: URI
+) : BoardPage(
+    driver = driver,
+    uri = uri
+) {
+    abstract fun goToBacklog(): SprintBoardComponent
+    abstract fun goToActiveSprint(): SprintPage

```

Comment:
I really like how these abstractions turned out. Scrum-specific features show up only in Scrum-specific objects.
And it's all safe in compile-time.